src/Model/State.re

```diff
@@ -54,6 +54,7 @@ type t = {
   pane: Pane.t,
   searchPane: Feature_Search.model,
   focus: Focus.stack,
+  modal: option(Modal.t),
```

Comment:
We might want to combine the modal and context menu model, so that a modal and context menu can't both be visible at the same time. Have to merge the context menu PR before doing that though.

Reply:
Cool. I think the context menu should be unblocked now (https://github.com/onivim/oni2/pull/1232 brought in the `boxShadow` styling)